lib/src/settings_section.dart

```diff
@@ -25,12 +25,13 @@ class SettingsSection extends AbstractSection {
   @override
   Widget build(BuildContext context) {
     final platform = Theme.of(context).platform;
-    if (platform.isIOS(context))
+    if (platform.isIOS(context)) {
       return iosSection();
-    else if (platform.isAndroid(context))
-      return androidSection(context);
-    else
+    }
+    if (platform.isAndroid(context)) {
       return androidSection(context);
+    }
+    return androidSection(context);
```

Comment:
I think we can make it more simple. Since the only platform with a different rendition is the iOS and all other platforms will use the `androidSection`, we can simplify it as:
```dart
if (platform.isIOS(context)) {
    return iosSection(context);
} else {
    // This takes care of Android and all other platforms
    return androidSection(context);
}
```

Reply:
Sure thing, just wanted to keep the same logic as before. I'll adjust.